### PR TITLE
Sam edits

### PR DESCRIPTION
**Pull request by sammurraytuesta**
Update Webpage and add content like blog posts, images, and about sections to add personality to the site!

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,17 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFunFact() {
+  // adjusted from greetings to facts about me!
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js]: Why don't you put this in the documentation comment above?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,17 @@
...
+  // functionality is the same as addRandomGreeting(),
+  // so content in array is replaced while the basic functionality remains the same
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js]: Google comment style suggests that you should always use complete sentences with capital letters at the start and period at the end of sentences.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -15,14 +15,17 @@
...
+  const facts =
+      ['I play the cello!', '¡Soy peruana y americana!', 'I love Marvel!', '¡Me encanta dibujar!', 'I am from Seattle!', 'I love watching basketball!'];
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js:20]: Nice fun facts. 🙂 (This is not an actionable comment.)

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -15,14 +15,17 @@
...
 
   // Pick a random greeting.
```
Comment (rjkroege) [inline portfolio/src/main/webapp/script.js]: Change greeting to fact? Comments and code should be in sync.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -7,3 +7,7 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.main-image{
+    width: 650px;
+}
```
Comment (rjkroege) [inline portfolio/src/main/webapp/style.css]: Try to end files with a newline.

**Review by rjkroege — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -10,14 +10,60 @@
...
       <h1>Samantha's Portfolio</h1>
       <p>This is my portfolio.</p>
       <p>Hello! My name is Samantha Murray Tuesta and I am a second-year undergraduate student at Western Washington University with a passion for all things computer science! Welcome to my portfolio!</p>
       <p>Click <a href="/hello">here</a> to see content created by server code.</p>
       <p>Click here to learn a fun fact about me:</p>
       <button onclick="addFunFact()">Fact</button>
       <div id="greeting-container"></div>
+      <img class="profile-pic" src="images/profilePicture.JPG"/>      
+      <h3>About Me</h3>
+      <p>I am a first generation college student, in my second year studying Computer Science and Internet Studies. I am currently based in Seattle, WA, and attend college in Bellingham!
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: Please line break text inside block elements.

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -10,14 +10,60 @@
...
+         <br><br>At my institution, Western Washington University, I am engaged in research specializing in digital privacy and its portrayal in the media. My work is aimed at creating a digital web game to engage students of all ages and teach the importance of digital privacy and how we go about advocating for those impacted by breaches in that privacy.
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: You are using `<br>` elements to control the visual appearance? I think that this runs contrary to the aspirational goal of HTML/CSS in that tags should express document semantic structure while CSS should control how it's rendered to the page. Consider using a `<p>` element for each block and having a CSS class that controls spacing?

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -10,14 +10,60 @@
...
+         <br><br>Some of my favorite activites include playing piano and cello in my free time, reading, drawing, and designing digital web pages! I first fell in love with computer science through learning HTML and enjoy incorporating that love in my research and studies!
+         <br><br>Growing up, I often visited my mother's hometown of Chiclayo, Peru and consider it my second home. I have always grown up with a love of culture and hope to one day explore other cultures to broaden my world-view!  
+      </p>
+      <h3>Projects</h3>
+      <ul>
+          <li>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: This seems like a bit of an abuse of the intent of the `<ul>` tag to me. We can discuss later.

File: portfolio/src/main/webapp/index.html [hunk 9]
```diff
@@ -59,11 +59,19 @@ <h3>Socials</h3>
       <p>GitLab: <a href="https://gitlab.com/sammurraytuesta">https://gitlab.com/sammurraytuesta</a></p>
       <h3>Gallery</h3>
       <img class="image" src="images/mini-me-hose.jpg"/>
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: `<figure>` and `<figcaption>` are specifically intended for this use case of having an image and a associated caption. Consider using them instead.

**Review by rjkroege — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/js/lightbox.js [hunk 10]
```diff
@@ -0,0 +1,4 @@
+
+$('document').ready(function () {
+    baguetteBox.run('.baguetteBoxOne');
+});
```
Comment (rjkroege) [inline portfolio/src/main/webapp/js/lightbox.js]: add trailing new line

**Review by rjkroege — CHANGES_REQUESTED** [review]
Why did you add the carousel to your in-progress PR? I would like you to:

* finish your previous PR by responding to my comments
* merge that
* and then create a new PR for adding the carousel

Also: did you need to check-in all of owl? Are there smaller choices? Can you add the third-party code (owl) in a separate PR from the code that uses it?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/script.js]: Comment is now added to the doc comment above!

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/script.js]: Perfect! Thank you! I have changed all comments to complete sentences! Let me know if I missed any! Thank you!

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/script.js:20]: Thank you! :)

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/script.js]: Greeting is now changed to fact. This js file should now be consistent!

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/style.css]: Noted! Thank you!

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: Elements are now spaced accordingly!

File: portfolio/src/main/webapp/index.html (hunk 7 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: The HTML has now been changed to "p" elements with CSS styling to adjust its proximity to other elements on the page!

File: portfolio/src/main/webapp/index.html (hunk 8 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: Yes I agree! I have changed this to normal paragraph blocks, and incorporated the use of the ul tag in listing off my socials!

File: portfolio/src/main/webapp/index.html (hunk 9 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: Thank you! I have made that change and learned something new!

File: portfolio/src/main/webapp/js/lightbox.js (hunk 10 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/js/lightbox.js]: This will be updated in the design-edits PR! Thank you!

**Comment by sammurraytuesta** [comment]
I have addressed all requested changes in my most recent commit to sam-edits and moved the new design choices and owl.carousel code to their own branches to later be submitted for PRs. As of right now, merging is still blocked and it still says there is one change requested, does that mean I should re-request review or is there something that I am missing? Thank you!

File: portfolio/src/main/webapp/css/style.css [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+    letter-spacing: 2vw;
+    font-size: 20vh;
+    padding: 2vh;
+    margin: 0;
+    font-family: 'Fascinate', cursive;
+    color: var(--pink);
+}
+
+.profile-pic{
+    fill: url(#pattern);
```
Comment (rjkroege) [inline portfolio/src/main/webapp/css/style.css]: I'm not clear on what this does. There must be a an SVG with an id of `pattern`?

File: portfolio/src/main/webapp/css/style.css [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+    letter-spacing: 2px;
+    font-size: 2.5vh;
+}
+
+.about-me .first-paragraph{
+    margin-top: 10%;
+}
+
+
+.owl-carousel{
```
Comment (rjkroege) [inline portfolio/src/main/webapp/css/style.css]: You are mixing up your dependencies here a bit in that some of this styling is only useful if you are already using the carousel yes?

File: portfolio/src/main/webapp/index.html [hunk 13]
```diff
@@ -1,77 +1,194 @@
...
+        <li><a href="#home" class="scroll">Home</a></li>
+        <li><a href="#about" class="scroll">About</a></li>
+        <li><a href="#projects" class="scroll">Projects</a></li>
+        <li><a href="#social" class="scroll">Social</a></li>
+      </ul>
+    </nav>
+  </div>
+
+  <section id="home">
+    <svg class="home-blob" viewBox="0 0 200 200" xmlns="http://www.w3.org/2000/svg">
```
Comment (rjkroege) [inline portfolio/src/main/webapp/index.html]: How did you generate the svg?

**Comment by rjkroege** [comment]
I was hoping that you'd separate the review response to your first PR and make a new PR for adding OWL support. This PR still adds a lot of OWL-related functionality. Can you divide this up more:

* *just* the review responses from the first PR
* a different PR where you add the carousel library
* a different PR where you modify your code to take advantage of the carousel?

**Review by rjkroege — APPROVED** [review]

File: portfolio/src/main/webapp/css/style.css (hunk 11 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/css/style.css]: yeah! So basically the image has the id pattern and I substitute the image in as the url for it to populate the inner area of the svg line!

File: portfolio/src/main/webapp/css/style.css (hunk 12 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/css/style.css]: I see what you are saying! Thank you!

File: portfolio/src/main/webapp/index.html (hunk 13 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/index.html]: I used a website known as svg blob generator to generate an svg line based on the shape I created using the tools on the page!

**Pull request merged**